pkgs/by-name/lu/luna-base/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  autoPatchelfHook,
+  fetchFromGitHub,
+  fftw,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "luna-base";
+  version = "1.0.0";
+  enableParallelBuilding = true;
+
+  src = fetchFromGitHub {
+    owner = "remnrem";
+    repo = "luna-base";
+    rev = "v${version}";
+    hash = "sha256-IWftBR1rU5yejdmngg6eFrLe/Pyfq/Qok/fq/cXyKX8=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    fftw
+  ];
+
```

```suggestion
  makeFlags = [
    "CXX=${stdenv.cc.targetPrefix}c++"
    "LD=${stdenv.cc.targetPrefix}ld"
  ];
```
Otherwise the package will not cross-compile, or build with a compiler other than GCC. (e.g. `luna-base.override { stdenv = clangStdenv; }`)